libs/tkrng/test/conanfile.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019-2022 Cambridge Quantum Computing
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from conans import ConanFile, CMake
+import platform
+
+
+class TestTkrngConan(ConanFile):
+    name = "test-tkrng"
+    version = "0.1.0"
+    license = "Apache 2"
+    url = "https://github.com/CQCL/tket"
+    description = "Unit tests for tkrng"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"with_coverage": [True, False]}
+    default_options = {"with_coverage": False}
+    generators = "cmake"
+    exports_sources = "*"
+    requires = ["tkrng/0.1.1", "catch2/3.0.1"]
```

Comment:
I do not think we should do that, even if it were possible -- part of the point is to know exactly which version you are using; otherwise expected behaviour may change suddenly under your feet.

Reply:
Okay, I was more thinking about keeping changes simple, but that is a good reason not to do that.